HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -889,6 +889,138 @@ copymethod:: SimpleNumber -atanh
 copymethod:: SimpleNumber -owensT
 
 
+subsection:: Testing
+
+method:: ==
+
+Returns true if each element is equal with its corresponding element in the receiver. See link::Classes/Object#Equality and Identity::.
+
+Discussion::
+Evaluate this code first to compare equality test and identity test:
+code::
+~array_a = [1, 2]; ~array_b = [1, 2]
+::
+
+list::
+## Equality:
+code::
+~array_a == ~array_b // checking for equality
+::
+teletype::
+-> true
+::
+It first checks if the two objects are identical, then they're not in the same class, then their sizes aren't the same, then there's an unequal element. If they are equal, then their emphasis::hash table indices:: are equal and identical:
```

Comment:
What are 'hash table indices'?
```
a = [1, 2]
b = [1, 2]

c = ( a : 10, b : 20 )

c[a] // 10
c[b] // 20
```

Reply:
whoa!... after 20 years of SuperCollider I haven't been aware of this syntax subtlety:
```supercollider
// compare with code from previous post
a = [1, 2]
b = [1, 2]

c = ( a: 10, b: 20 ) // no space between keys and colons!

c[a] // nil
c[\a] // 10
c.a // 10
```
... I think this would be worth mentioning in the Event helpfile...

Reply:
subtle.

Reply:
https://github.com/supercollider/supercollider/issues/6386

It's a good first issue.

Reply:
@JordanHendersonMusic
Still not sure what a "hashed table index" is? Can I mark it as solved or do I need to remove/change it?

Reply:
> How about this?
It first checks if the two objects are identical, then they're not in the same class, then their sizes aren't the same, then there's an unequal element. If they are equal, then their...

This is all implementation and should be avoided. Just say, checks the contents of the string to see if they are the same.